winterfell/webapp/src/js/formCtrl.js

```diff
@@ -9,6 +9,29 @@ app.controller("formCtrl", ['$scope', 'formRenderingService', function($scope, f
     });
 
     $scope.render = function() {
-        formRenderingService.render('VBA-21-0966-ARE', [{fieldName: 'veteran_first_name', fieldValue: 'jeff'}]);
-    }
+        var out = [];
+        for (var k in $scope.model) {
+            if ($scope.model.hasOwnProperty(k)) {
+                out.push({fieldName: k, fieldValue: $scope.model[k]})
+            }
+        }
+        formRenderingService.render('VBA-21-0966-ARE', out);
```

Comment:
it would be good practice to start placing constants like Form names, titles, fields, etc. into its own "class".
Feel free to use whatever makes sense for you.
I have a strings.js (containing military ranks -> text, not that we really need it, but it's there) in the `libs` folder and its gets treated like a library so you can to import it elsewhere.